web/pdf_find_controller.js

```diff
@@ -203,15 +203,32 @@ var PDFFindController = (function PDFFindControllerClosure() {
         // If the page is selected, scroll the page into view, which triggers
         // rendering the page, which adds the textLayer. Once the textLayer is
         // build, it will scroll onto the selected match.
-        this.pdfViewer.scrollPageIntoView(index + 1);
+          if (PDFJS.multiple === undefined) {
+              this.pdfViewer.scrollPageIntoView(index + 1);
+          }
       }
 
       if (page.textLayer) {
         page.textLayer.updateMatches();
       }
+
+      //scroll match into view
+      if (PDFJS.multiple !== undefined) {
+          try {
+              var d = document.getElementsByClassName('highlight');
+              if (d.length > 0) {
+                  var vc = document.getElementById('viewerContainer');
+                  if (vc.scrollTop < 40) {
```

Comment:
Is the `40` here supposed to be toolbar height? That should probably not be set as a constant here.